README.md

```diff
@@ -19,36 +15,37 @@ Documentation for Puppet and related projects can be found online at the
 
 ## Installation
 
-The best way to run Puppet is with [Puppet Enterprise (PE)](https://puppet.com/products/puppet-enterprise/),
-which also includes orchestration features, a web console, and professional support.
-The PE documentation is [available here.](https://puppet.com/docs/pe/latest)
-
-To install an open source release of Puppet,
-[see the installation guide on the docs site.](https://puppet.com/docs/puppet/latest/installing_and_upgrading.html)
+To install OpenVox,
+[see our installation guide.](https://voxpupuli.org/openvox/install/) and [quickstart page](https://voxpupuli.org/openvox/quickstart/).
 
-If you need to run Puppet from source as a tester or developer,
+If you need to run OpenVox from source as a tester or developer,
 see the [Quick Start to Developing on Puppet](docs/quickstart.md) guide.
 
 ## Developing and Contributing
 
 We'd love to get contributions from you! For a quick guide to getting your
 system setup for developing, take a look at our [Quickstart
 Guide](https://github.com/puppetlabs/puppet/blob/main/docs/quickstart.md). Once you are up and running, take a look at the
-[Contribution Documents](https://github.com/puppetlabs/.github/blob/main/CONTRIBUTING.md) to see how to get your changes merged
+[Contribution Documents](https://github.com/OpenVoxProject/.github/blob/main/CONTRIBUTING.md) to see how to get your changes merged
 in.
 
 For more complete docs on developing with Puppet, take a look at the
 rest of the [developer documents](https://github.com/puppetlabs/puppet/blob/main/docs/index.md).
 
 ## Licensing
 
-See [LICENSE](https://github.com/puppetlabs/puppet/blob/main/LICENSE) file. Puppet is licensed by Puppet, Inc. under the Apache license. Puppet, Inc. can be contacted at: info@puppet.com
+See [LICENSE](https://github.com/puppetlabs/puppet/blob/main/LICENSE) file. OpenVox is licensed by Vox Pupuli as a community maintained
+implementation of Puppet. Vox Pupuli can be contacted at: pmc@voxpupuli.org.
```

Comment:
maybe also mention voxpupuli@groups.io? That's the public mailing list, not just the PMC.